osu.Game/Graphics/Carousel/Carousel.cs

```diff
@@ -514,6 +518,29 @@ private void traverseGroupSelection(int direction)
 
         #endregion
 
+        #region Audio
+
+        private Sample? sampleKeyboardTraversal;
+
+        private double audioFeedbackLastPlaybackTime;
+
+        [BackgroundDependencyLoader]
+        private void load(AudioManager audio)
+        {
+            sampleKeyboardTraversal = audio.Samples.Get(@"UI/button-hover");
```

Comment:
I get it's a bit ugly to have this split across two classes. If it's seen as an issue, I'd probably want to move *more* to the base class, then override in `BeatmapCarousel` to get per-panel-type behaviour (ie ensure that if we ever use `Carousel` elsewhere it will still have some basic sounds setup).

Reply:
Also the sample here was arbitrarily selected to meld well with the hover sounds when the mouse is hovering the carousel and the user is keyboard selecting:


https://github.com/user-attachments/assets/c8ba6b9c-4d76-4c99-ba37-053011888697